ingestion/src/metadata/ingestion/ometa/mixins/custom_property_mixin.py

```diff
@@ -76,3 +80,12 @@ def get_property_type_ref(self, data_type: CustomPropertyDataTypes) -> PropertyT
         """
         custom_property_type = self.get_custom_property_type(data_type=data_type)
         return PropertyType(EntityReference(id=custom_property_type.id, type="type"))
+
+    def get_entity_custom_properties(self, entity_type: Type[T]) -> Optional[List]:
+        """
+        Get all the custom properties of an entity
+        """
+        resp = self.client.get(
+            f"/metadata/types/name/{ENTITY_REFERENCE_TYPE_MAP.get(entity_type.__name__)}?fields=customProperties"
+        )
+        return resp.get("customProperties")
```

Comment:
where are we using this? if this is just for the sdk then we should include test to demonstrate how to use it